zulip/integrations/bridge_with_matrix/matrix_bridge.py

```diff
@@ -77,7 +77,7 @@ def _matrix_to_zulip(room: Any, event: Dict[str, Any]) -> None:
         """
         content = get_message_content_from_event(event, no_noise)
 
-        zulip_bot_user = "@{}:matrix.org".format(matrix_config["username"])
```

Comment:
This needs to be just `zulip_bot_user = matrix_config["username"]`. No need for format.